### PR TITLE
New Version: VSDC.Editor 7.1.1.392

### PR DESCRIPTION
**Pull request by Trenly**
* Resolves #58883

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58900)

### CODE REVIEW

File: manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.installer.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-- Architecture: x86
-  InstallerUrl: https://downloads.videosoftdev.com/video_tools/video_editor_x32.exe
-  InstallerSha256: 15C53C36CED9DEB3D93B2AABD740C2F19FF5D081BACED2FE63B232028EBDE2E7
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390
```
Comment (ItzLevvie) [inline manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.installer.yaml:5]:
```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.

File: manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.locale.en-US.yaml [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-# Agreements: 
-# ReleaseNotes: 
-# ReleaseNotesUrl: 
-ManifestType: defaultLocale
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390
```
Comment (ItzLevvie) [inline manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.

File: manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-PackageIdentifier: VSDC.Editor
-PackageVersion: 6.9.5.382
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390
```
Comment (ItzLevvie) [inline manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.yaml:5]:
```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Closing as version already present and merged

**Pull request closed without merging**